framework/include/cppmicroservices/detail/TrackedService.tpp

```diff
@@ -109,44 +103,51 @@ void TrackedService<S,TTT>::ServiceChanged(const ServiceEvent& event)
       }
       break;
     }
-  case ServiceEvent::SERVICE_MODIFIED_ENDMATCH :
-  case ServiceEvent::SERVICE_UNREGISTERING :
-    this->Untrack(reference, event);
-    /*
+    case ServiceEvent::SERVICE_MODIFIED_ENDMATCH:
+    case ServiceEvent::SERVICE_UNREGISTERING:
+      this->Untrack(reference, event);
+      /*
      * If the customizer throws an unchecked exception, it is
      * safe to let it propagate
      */
-    break;
+      break;
   }
 }
 
 template<class S, class TTT>
-void TrackedService<S,TTT>::Modified()
+void TrackedService<S, TTT>::Modified()
 {
   Superclass::Modified(); /* increment the modification count */
   serviceTracker->d->Modified();
 }
 
 template<class S, class TTT>
-std::shared_ptr<typename TrackedService<S,TTT>::TrackedParamType>
-TrackedService<S,TTT>::CustomizerAdding(ServiceReference<S> item,
-                                        const ServiceEvent& /*related*/)
+std::optional<
+  std::shared_ptr<typename TrackedService<S, TTT>::TrackedParamType>>
+TrackedService<S, TTT>::CustomizerAdding(ServiceReference<S> item,
+                                         const ServiceEvent& /*related*/)
 {
-  return customizer->AddingService(item);
+  auto value = customizer->AddingService(item);
```

Comment:
Rename variable to be more descriptive

Reply:
Changed `value` to `serviceObjectPointer`, and added a comment to better explain what the return statement is doing.